bin/docker-init.sh

```diff
@@ -162,6 +166,10 @@ if [ "$WILL_CREATE_CLUSTER" = true ]; then
         echo "[status] setting up cluster with local Docker registry support"
         KIND_MANIFEST=/usr/share/kubemacs/kind-cluster+registry.yaml
     fi
+    if [ "$KIND_LOCAL_NETBOOT_ENABLE" = true ]; then
```

Comment:
Use `elif` or `case` statement set.

If local registry is false and netboot is true, then do this.